collision.go

```diff
@@ -134,9 +136,13 @@ func (cs *CollisionSystem) UpdateEntity(entity *ecs.Entity, dt float32) {
 				}
 
 				Mailbox.Dispatch(CollisionMessage{Entity: entity, To: other})
+				collision.Collides = true
```

Comment:
Yep seen that yesterday night after pushing. But I was too tired to cleanup. But I agree.